aten/src/ATen/function_wrapper.py

```diff
@@ -292,6 +302,9 @@ def __getitem__(self, x):
             return r
         return self.parent[x]
 
+    def __setitem__(self, key, value):
+        return self.base.__setitem__(key, value)
```

Comment:
do we really need this?  It's not obvious that it should set the base vs the parent to me, and I think it generally has helped code organization to set up all the dictionaries before hand, generate the nested dict, then do the code gen.

Reply:
We don't. It's a leftover from a refactor that I rolled back. Will remove